.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Release Docs
```

Comment:
@alebedev87 Have you looked at this new github workflow? It looks like it auto-generates docs and commits them back to the repo. I think it only happens if you create a tag with a version number, so if we don't tag our versions like that it probably won't matter, but it may be adding more commits that will make the next rebase a bit more work.

I think it's probably fine to leave this alone, but if it starts getting triggered, we may want to disable it like we did with dependabot

Reply:
Good catch! I didn't have a look at the github workflow. Yes, indeed it might have been a problem especially if we would make use of tags in this repository. Will have to see whether it'll give us some noise.